### PR TITLE
Howstheeweather it's a simple web app to get the current weather conditions in a city

### PR DESCRIPTION
**Pull request by carlosprietobarron**
This web app was created using the API facilitated by  https://openweathermap.org/api and was developed using JavaScript and bootstrap, as well as HTML and CSS.

The changes to merge are:

Module and object to handle the Request and Response to the API.

Module and object to request the current location of the user

Module to display the results of the request in the browser.

### CODE REVIEW

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

Your project is working as it should.

However, inside your package.json, add common commands like `build` and `development` inside scripts object.

* [ ] In your package.json
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  }
```
When you are done, submit another form for a code.

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+const initialize = () => {
+  domcon.createIcon();
+  domcon.setBtnEvent();
+  domcon.setToggle();
+};
+
+initialize();
+geoLib.geoLocal();
+// getDATA(cityName, getWeather.getWeaNow);
```
Comment (uche-inyama) [inline src/index.js]: Delete commented code.

File: src/video.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const videoLooper = (function vd() {
+  function looper(elem) {
+    if (elem.currentTarget.currentTime >= 4) {
+      elem.currentTarget.currentTime = 1;
+    // elem.play();
```
Comment (uche-inyama) [inline src/video.js]: Delete commented code.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required.

This comment is a continuation of the first review.

* [ ] Remove commented codes.

**Review by MarioDena — APPROVED** [review]
Hello! Great work implementing the requested changes! 🎖️ 

Project Approved 🟢 

Cheers! 😄

**Pull request merged**